api/setup.py

```diff
@@ -60,21 +59,27 @@ def recur_expand(target_root, dir):
     entry_points={
         'console_scripts': [
             'zulip-send=zulip.send:main',
+            'zulip-bot-server=zulip.bot_server:main',
         ],
     },
+    test_suite='tests',
 )  # type: Dict[str, Any]
 
 setuptools_info = dict(
     install_requires=['requests>=0.12.1',
                       'simplejson',
                       'six',
                       'typing>=3.5.2.2',
+                      'flask>=0.12.2',
+                      'mock>=2.0.0',
                       ],
 )
 
 try:
-    from setuptools import setup
+    from setuptools import setup, find_packages
     package_info.update(setuptools_info)
+    package_info['packages'] = find_packages()
+
```

Comment:
I think we should use `find_package`'s `exclude` parameter to exclude the `tests/` from the package installation (the tests are for development purposes only). See [this](http://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages) for more info. :)